### PR TITLE
Updated Czech (CZ) 🇨🇿 [i18n] Translation

### LINKED ISSUE #4655: [i18n] Translation Czech (CZ) 🇨🇿 - Hacktoberfest Task
Dear Hacktoberfest-Community,

please help us to improve our Czech translations.

While we develop our [FreshRSS ](https://freshrss.org/) app we cannot support the translations in all 15+ languages. 

## About the FreshRSS app
FreshRSS is a free, self-hostable feed aggregator licenced by [GNU3.0](https://github.com/FreshRSS/FreshRSS/blob/edge/LICENSE.txt). These days we celebrate the 10th birthday!

## Actual Status
Right now about 90% of our text is translated into Czech.

## How to support us
How to do it:
1) it is very simple
2) all files are stored in `https://github.com/FreshRSS/FreshRSS/tree/edge/app/i18n/cz` ([Link](https://github.com/FreshRSS/FreshRSS/tree/edge/app/i18n/cz))
3) each untranslated text has an inline comment `//  TODO`
4) When the text is translated, just delete the comment
5) If the English term fits as well, please change the comment from `//  TODO` to `//  IGNORE`
6) open a Pull Request and link to this ticket

## Thanks
Thanks a lot. It will help our very international community of users and developers. Your submitted PRs will be published with our next release version 1.20.1 or 1.21.0

### PR DESCRIPTION
**Pull request by MayMeow**
Update missing translation on missing Czech strings

Closes #4655 

Changes proposed in this pull request:

- Updating Czech translation

How to test the feature manually:

1. by switching application to Czech language

### CODE REVIEW

**Comment by MayMeow** [comment]
https://github.com/FreshRSS/FreshRSS/blob/79e3b99ae3b9df4bb02957838fc6b62491ab0bd2/app/i18n/cz/sub.php#L78

`Web scraping` is same here in (Slovakia/Czech) we don't translate it. When you translate it `Vyškrabávání z webu` in CZ and `Škrabanie webových stránok` and in SK. But I think nobody will understand about what you are talking :) (It sounds kind a funny. Trust me 😃 )

**Comment by math-GH** [comment]
> https://github.com/FreshRSS/FreshRSS/blob/79e3b99ae3b9df4bb02957838fc6b62491ab0bd2/app/i18n/cz/sub.php#L78
> 
> `Web scraping` is same here in (Slovakia/Czech) we don't translate it. When you translate it `Vyškrabávání z webu` in CZ and `Škrabanie webových stránok` and in SK. But I think nobody will understand about what you are talking :) (It sounds kind a funny. Trust me 😃 )

I changed the `TODO` comment into `IGNORE`.

Could you please check the few left `TODO` and `DIRTY` lines?

File: app/i18n/cz/conf.php [hunk 1]
```diff
@@ -136,31 +136,31 @@
...
+				'both' => 'V záhlaví a zápatí',
+				'footer' => 'V zápatí',
+				'header' => 'V záhlaví',
+				'none' => 'Žádné',
 			),
 			'tags_max' => array(
-				'_' => 'Max number of tags shown',	// TODO
-				'help' => '0 means: show all tags and do not collapse them',	// TODO
+				'_' => 'Max number of tags shown',
+				'help' => '0 means: show all tags and do not collapse them',
```
Comment (math-GH) [inline app/i18n/cz/conf.php]: it is still an TODO

File: app/i18n/cz/admin.php [hunk 2]
```diff
@@ -170,18 +170,18 @@
...
 			'select' => array(
-				'label' => 'Registration form',	// TODO
+				'label' => 'Registrační formulář',
 				'option' => array(
-					'noform' => 'Disabled: No registration form',	// TODO
-					'nolimit' => 'Enabled: No limit of accounts',	// TODO
-					'setaccountsnumber' => 'Set max. number of accounts',	// TODO
+					'noform' => 'Zakazáno: Žádný registrační formulář',
+					'nolimit' => 'Povoleno: Bez omezení počtu účtů',
+					'setaccountsnumber' => 'Nastavit max. počet účtů',	// TODO also it can be: Nastavit maximální počet účtů
```
Comment (math-GH) [inline app/i18n/cz/admin.php]: A decision is needed here ;)

**Comment by math-GH** [comment]
Awesome. That is a great improvement for FreshRSS.

Please add yourself on our [Credits ](https://github.com/FreshRSS/FreshRSS/blob/edge/CREDITS.md) list :)

File: app/i18n/cz/admin.php (hunk 2 above)
Comment (MayMeow) [reply app/i18n/cz/admin.php]: I think `Nastavit maximální počet účtů` willl be better, if there is not an limit on length.

Comment (math-GH) [reply app/i18n/cz/admin.php]: It is enough space here :) 
grafik

Comment (MayMeow) [reply app/i18n/cz/admin.php]: Updated it to `Nastavit maximální počet účtů`

File: app/i18n/cz/conf.php (hunk 1 above)
Comment (MayMeow) [reply app/i18n/cz/conf.php]: Fixed this and also found another missing in `sub.php` so i updated this one aswell.

File: app/i18n/cz/sub.php [hunk 3]
```diff
@@ -126,8 +126,8 @@
 			'reload_articles_help' => 'Znovu načíst články a získat úplný obsah, pokud je definován přepínač.',	// DIRTY
```
Comment (math-GH) [inline app/i18n/cz/sub.php]: Could you please check this line? It is marked as "Dirty". I am not sure why. Is it a correct translation of `Reload that many articles and fetch complete content if a selector is defined.` ?

A `selector `is:
grafik

File: app/i18n/cz/sub.php [hunk 4]
```diff
@@ -73,51 +73,51 @@
...
-					'_' => 'item date',	// TODO
-					'help' => 'The result will be parsed by <a href="https://php.net/strtotime" target="_blank"><code>strtotime()</code></a>',	// TODO
+					'_' => 'datum položky',
+					'help' => 'Výsledek bude zpracován pomocí <a href="https://php.net/strtotime" target="_blank"><code>strtotime()</code></a>',
 				),
 				'item_title' => array(
-					'_' => 'item title',	// TODO
-					'help' => 'Use in particular the <a href="https://developer.mozilla.org/docs/Web/XPath/Axes" target="_blank">XPath axis</a> <code>descendant::</code> like <code>descendant::h2</code>',	// TODO
+					'_' => 'název položky',
+					'help' => 'Použijte zejména <a href="https://developer.mozilla.org/docs/Web/XPath/Axes" target="_blank">XPath axis</a> <code>descendant::</code> like <code>descendant::h2</code>',	// DIRTY
```
Comment (math-GH) [inline app/i18n/cz/sub.php]: This line is marked as `DIRTY` because the word `like` is not yet translated

File: app/i18n/cz/sub.php (hunk 3 above)
Comment (MayMeow) [reply app/i18n/cz/sub.php]: `přepínač` is `switch` IMO for selector is better translation `selektor` or `volič`, but I lean more towards the first option

Comment (math-GH) [reply app/i18n/cz/sub.php]: Do you want to change it or keep it as it is?

Comment (MayMeow) [reply app/i18n/cz/sub.php]: Sorry for delay, i updated it after discuss with colleague

**Pull request merged**